config/rbac/role.yaml

```diff
@@ -77,6 +77,14 @@ rules:
   verbs:
   - patch
   - update
+- apiGroups:
```

Comment:
You need to specify this via the kube-builder annotation. Please refer to the following 
https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/f861a8e583af3033435885acb9c04b6a32dab84f/controllers/elbv2/targetgroupbinding_controller.go#L83

Once you specify the kubebuilder annotation, make generate will automatically generate this file